spec/controllers/api/enterprises_controller_spec.rb

```diff
@@ -21,15 +21,33 @@ module Api
       end
 
       describe "creating an enterprise" do
-        let(:australia) { Spree::Country.find_by_name('Australia') }
-        let(:new_enterprise_params) { {enterprise: {name: 'name', email: 'email@example.com', address_attributes: {address1: '123 Abc Street', city: 'Northcote', zipcode: '3070', state_id: australia.states.first, country_id: australia.id } } } }
+        let(:address) { build(:address) }
+
+        before :each do
+          @new_enterprise_params = {
```

Comment:
you can use `let(:new_enterprise_params)` here, right? no need for instance variables